Tasks/JavaToolInstallerV0/Tests/L0.ts

```diff
@@ -38,14 +38,4 @@ describe('JavaToolInstaller L0 Suite', function () {
         assert(testRunner.failed, 'task should have failed');
         done();
     });
-
-    it('should run succesfully when extracting zip file', (done) => {
```

Comment:
While we are touching this task, can we change this to `function ()`?  (I am going to fix this in all our tasks eventually).

We also don't need to pass `done` -- this is just for validating callbacks.  So just pass `function ()` and delete the calls to `done()`.